File: src/main/java/core/example/java/Movie.java
package core.example.java;

public class Movie {
    private String name;
    private String director;
    private String stuntMaster;
    private String numberOfArtists;
    private String releaseDate;

    @Override
    public String toString() {
        return "Movie{" +
                "name='" + name + '\'' +
                ", director='" + director + '\'' +
                ", stuntMaster='" + stuntMaster + '\'' +
                ", numberOfArtists='" + numberOfArtists + '\'' +
                ", releaseDate='" + releaseDate + '\'' +
                '}';
    }

    public Movie(String name, String director, String stuntMaster, String numberOfArtists, String releaseDate) {
        this.name = name;
        this.director = director;
        this.stuntMaster = stuntMaster;
        this.numberOfArtists = numberOfArtists;
        this.releaseDate = releaseDate;
    }
}

File: src/main/java/core/example/kotlin/Movie.kt
package core.example.kotlin

class Movie (private val name:String, private val director:String,
            private val stuntMaster:String,private val numberOfArtists:String,
            private val releaseDate:String) {
    override fun toString(): String {
        return "Movie(name='$name', director='$director', stuntMaster='$stuntMaster', numberOfArtists='$numberOfArtists', releaseDate='$releaseDate')"
    }
}
